Class/C-Objects-2/exercise2.js

```diff
@@ -25,14 +25,24 @@ let kinningParkHouse = {
   */
   
   // returns the full name (first name + last name) of the owner of the house
-  function getOwnerFullName(house) {}
+  function getOwnerFullName(house) {
+    return house.currentOwner.firstName + " " + house.currentOwner.lastName;
+  }
   
   // returns an array of the owners' email addresses of the two houses
   // search for "javascript array concat" to remember how to join two arrays
-  function getEmailAddresses(house1, house2) {}
+  function getEmailAddresses(house1, house2) {
+    return house1.currentOwner.email + ", " + house2.currentOwner.email;
+  }
   
   // returns the address for the cheapest house out of the two
-  function getCheapestAddress(house1, house2) {}
+  function getCheapestAddress(house1, house2) {
+    if(house1.price < house2.price){
+      return house1.address;
+    }else{
+      return house2.address;
+    }
+  }
   

```

Comment:
Hey Megumi, this is good.
Minor suggestion that might save you some code writing (so called "boiler plate"): be lazy at typing, nothing wrong with that, if you do it nicely and your solution as well readable and understandable by others.
For example, for choosing just a `min` value out of a list on numbers it's easy and nicely to use existing method `Math.min()`, please check out the MDN pages for this one.